openquakeplatform/openquakeplatform/static/js/risk_input_model_exposure.js

```diff
@@ -0,0 +1,437 @@
+/*
+   Copyright (c) 2015, GEM Foundation.
+
+      This program is free software: you can redistribute it and/or modify
+      it under the terms of the GNU Affero General Public License as
+      published by the Free Software Foundation, either version 3 of the
+      License, or (at your option) any later version.
+
+      This program is distributed in the hope that it will be useful,
+      but WITHOUT ANY WARRANTY; without even the implied warranty of
+      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+      GNU Affero General Public License for more details.
+
+      You should have received a copy of the GNU Affero General Public License
+      along with this program.  If not, see <https://www.gnu.org/licenses/agpl.html>.
+*/
+
+var table;
+var header;
+var showArea = false;
+
+$( document ).ready(function() {
+    updateTable();
+    $('#outputDiv').css('display', 'none');
+});
+
+$('#retrofittingSelect').hide();
+
+/////////////////////////////////////////////////////////
+// Manage the visibility of the perArea selection menu //
+/////////////////////////////////////////////////////////
+$('#perArea').hide();
+
+$('#defineCostStruc').change(function() {
+    var defineCost = $('#defineCostStruc').val();
+    if (defineCost == 'per_area') {
+        $('#perArea').show();
+        showArea = true;
+    }
+    if (defineCost != 'none') {
+        $('#retrofittingSelect').show();
+    } else {
+        $('#retrofittingSelect').hide();
+        // uncheck retrofitting
+        $('#retroChbx').attr('checked', false);
+    }
+});
+
+$('#defineCostNonStruc').change(function() {
+    var defineCost = $('#defineCostNonStruc').val();
+    if (defineCost == 'per_area') {
+        $('#perArea').show();
+        showArea = true;
+    }
+});
+
+$('#defineCostContent').change(function() {
+    var defineCost = $('#defineCostContent').val();
+    if (defineCost == 'per_area') {
+        $('#perArea').show();
+        showArea = true;
+    }
+});
+
+$('#defineCostBusiness').change(function() {
+    var defineCost = $('#defineCostBusiness').val();
+    if (defineCost == 'per_area') {
+        $('#perArea').show();
+        showArea = true;
+    }
+});
+
```

Comment:
Perhaps you could define a function similar to:
```javascript
function defineCost(costType) {
    var defineCost = $(costType).val();
    if (defineCost == 'per_area') {
        $('#perArea').show();
        showArea = true;
}
```
And call it in the 4 above cases